Dockerfile

```
@@ -9,11 +9,15 @@ FROM debian:buster AS build
 # Allow users to specify forked node, or specific commit
 # If not fallback to release branch (or alternatively just specify branch)
 ARG NODE_REPO=https://github.com/casper-network/casper-node.git
-ARG NODE_COMMIT=
 ARG CLIENT_REPO=https://github.com/casper-ecosystem/casper-client-rs.git
-ARG CLIENT_COMMIT=
-ARG NODE_GITBRANCH=release-1.5.6
+ARG SIDECAR_REPO=https://github.com/casper-network/casper-sidecar.git
+
+ARG NODE_COMMIT=
+ARG NODE_GITBRANCH=feat-2.0
 ARG CLIENT_GITBRANCH=release-2.0.0
```

`casper-client v2.0.0` is made for Casper 1.5.6. For 2.0 you can use:

```suggestion
ARG CLIENT_GITBRANCH=feat-track-node-2.0
```

---

Fixed in 686f53fce12a6fd2e7b7590430eaf8902c22f01e.